### ISSUE TITLE
Clarification on CodeContests Dataset Splits in CURE vs. DeepMind Original

### ISSUE THREAD

**Issue by yuzhu-cai** [Gen-Verse/CURE#3]
Hi,

I'm trying to understand the CodeContests dataset used in CURE. Specifically:

- What is the relationship between CURE's test set and DeepMind's original test/validation sets?
- Is the current test set derived/split from DeepMind's original training set?

Any details would be greatly appreciated. Thanks!

**Comment by yinjjiew** [comment]
Yes, we first load the huggingface dataset `deepmind/code_contests`, and select all tasks with difficulty level <= 2 in its training set. Then we randomly split it to obtain our 4.7k training set and 239 test set.
